pkgs/development/r-modules/generic-builder.nix

```diff
@@ -1,10 +1,14 @@
-{ stdenv, R, xvfb_run, utillinux }:
+{ stdenv, R, tzdata, gettext, gfortran, libcxx, xvfb_run, utillinux }:
 
 { name, buildInputs ? [], ... } @ attrs:
 
 stdenv.mkDerivation ({
-  buildInputs = buildInputs ++ [R] ++
-                stdenv.lib.optionals attrs.requireX [utillinux xvfb_run];
+  buildInputs = buildInputs ++ [ R tzdata gettext gfortran ] ++
```

Comment:
Why are `tzdata`, `gettext`, and `gfortran` added to `buildInputs` in the generic builder? Linux works fine without those packages in $PATH because records those paths in its internal config files. Does this not happpen on Darwin?